src/fidesops/service/masking/strategy/masking_strategy.py

```diff
@@ -13,10 +13,20 @@ class MaskingStrategy(ABC):
     """Abstract base class for masking strategies"""
 
     @abstractmethod
-    def mask(self, value: Optional[str]) -> Optional[str]:
+    def mask(self, value: Optional[str], request_id: Optional[str]) -> Optional[str]:
         """Used to mask the provided value"""
         pass
 
+    @staticmethod
+    @abstractmethod
+    def secrets_required() -> bool:
+        """Determines whether secrets are needed for specific masking strategy"""
+        pass
```

Comment:
`pass` isn't required when a docstring is present — we should keep this consistent across the codebase

Reply:
ok, I'll remove the `pass` statements from the rest of this class too